rmgpy/data/solvationTest.py

```diff
@@ -193,6 +193,12 @@ def test_radical_and_lone_pair_generation(self):
         solute_data = self.database.get_solute_data_from_groups(species)
         self.assertIsNotNone(solute_data)
 
+    def test_radical_solute_group(self):
+        """Test that the existing radical group is found for the radical species when using group additivity"""
+        species = Species(molecule=[Molecule(smiles='[OH]')])
+        solute_data = self.database.get_solute_data_from_groups(species)
+        self.assertTrue('radical' in solute_data.comment)
+
```

Comment:
Just a thought, should we add a test to check if we get the same (we shouldn't) Abraham solute parameters for OH radical and water? If we get the same values then we know something is wrong.

Reply:
I could add that test. I will just add an additional test for checking whether their A parameters are different